docs/Installation-macOS.md

```diff
@@ -25,3 +25,13 @@ All of the commands shown in the following instructions are meant to be run as a
 1. Run the installer:
 
 		bash -c "$(curl -fsSL https://raw.githubusercontent.com/theos/theos/master/bin/install-theos)"
+
+# Troubleshooting
```

Comment:
Given that this isn't exactly installation related, do we want to add a `Troubleshooting.md` page under Help.md where we can add common issues + fixes like this?

Reply:
Again, we can't assume users will read all the non-critical doc pages, because they don't

Reply:
This page isn't that large and I agree with Naville's comment. Resolving